.github/workflows/gradle.yml

```diff
@@ -39,20 +33,6 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Checkout the utils
```

Comment:
If I understand correctly with this change we only test our docker build tasks?  Would not be better to test both?

Reply:
You're right, I added it back in

I also added a summary so we can see how long things took to run

https://github.com/micronaut-projects/micronaut-crac-tests/actions/runs/3174048446

<img width="583" alt="image" src="https://user-images.githubusercontent.com/49317/193578215-19c3ce99-e785-41ac-8ad2-e4b89be2276c.png">

Reply:
This looks great thanks.